src/CraigslistAPI/CraigslistSearchComponent.css

```diff
@@ -0,0 +1,7 @@
+a {
+    color: lightgray;
+}
+
+a:hover {
+    color: grey;
+}
```

Comment:
Lets not change all `a` tags into light gray.

I would suggest adding a class/id to the `privacy policy` link element and using that as the CSS target